tests/Doctrine/Tests/DBAL/Functional/Schema/MySqlSchemaManagerTest.php

```diff
@@ -220,15 +220,13 @@ public function testColumnCharset(): void
     {
         $table = new Table('test_column_charset');
         $table->addColumn('id', 'integer');
-        $table->addColumn('no_charset', 'text');
         $table->addColumn('foo', 'text')->setPlatformOption('charset', 'ascii');
         $table->addColumn('bar', 'text')->setPlatformOption('charset', 'latin1');
         $this->schemaManager->dropAndCreateTable($table);
 
         $columns = $this->schemaManager->listTableColumns('test_column_charset');
 
         self::assertFalse($columns['id']->hasPlatformOption('charset'));
-        self::assertEquals('utf8', $columns['no_charset']->getPlatformOption('charset'));
```

Comment:
This assertion was removed in https://github.com/doctrine/dbal/pull/4644 as well. In this test, we don't care what charset the database uses by default.